java/gazelle/testdata/annotations/src/main/java/com/example/BUILD.want.res

```diff
@@ -0,0 +1,8 @@
+load("@rules_java//java:defs.bzl", "java_library")
+
+java_library(
+    name = "example",
+    srcs = ["Main.java"],
+    _java_packages = ["com.example"],
```

Comment:
Let's remove `_java_packages` from the resolve files - this is an internal implementation detail of the resolver, and the output is not part of the public API of the resolver

Reply:
Yeah, sorry. I forgot to re-generate golden files :D Done.